test/server/listener_manager_impl_test.cc

```diff
@@ -320,6 +333,7 @@ TEST_F(ListenerManagerImplTest, AddListenerAddressNotMatching) {
   ListenerHandle* listener_foo = expectListenerCreate(false);
   EXPECT_CALL(listener_factory_, createListenSocket(_, true));
   EXPECT_TRUE(manager_->addOrUpdateListener(*loader));
+  checkStats(1, 0, 0, 0, 1, 0);
```

Comment:
I can't make up my mind if this is actually a good suggestion or not but what the heck:

Call the method checkStatsAMRWAD(), at least it's always obvious which order the args are.